docs/examples/dali_plugin/README.md

```diff
@@ -0,0 +1,63 @@
+# DALI plugin with Triton
+
+This example shows how to use custom operations implemented for DALI
+(i.e. DALI plugins) with Triton. As an example, we use here a custom DALI
+operation that copies the input to the output. You can find this copy-plugin in
+[DALI documentation](https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/examples/custom_operations/custom_operator/create_a_custom_operator.html).
+
+### Step 1: Obtain the plugin lib
+
+Firstly, you need to code and build DALI plugin. This topic is
+out of scope of this tutorial. You can find all relevant info in
+[DALI documentation](https://docs.nvidia.com/deeplearning/dali/master-user-guide/docs/examples/custom_operations/custom_operator/create_a_custom_operator.html). All you need to do is to follow linked tutorial to obtain `.so` library with your custom plugin.
+
+### Step 2: Serialize DALI pipeline which uses your custom operation
+
+As always when using DALI with Triton, you need to serialize your DALI pipeline.
+Again, this topic is broadly discussed in the tutorial linked above. To summarize,
+you need to use `nvidia.dali.plugin_manager` to load your plugin and use it in
+your pipeline. `custom_copy_pipeline.py` file shows, how do to it:
+
+    plugin_manager.load_library('/path/to/libcustomcopy.so')
+    
+    @dali.pipeline_def(batch_size=3, num_threads=1, device_id=0)
+    def pipe():
+        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
+        cp = dali.fn.custom_copy(data)
+        return cp
+
+### Step 3: Inject plugin lib into tritonserver docker image
+
+The `.so` file you have built in the previous step contains the
+definition of your operation. Therefore, you need to inject it
+into your tritonserver image, so that DALI would be able to conduct
+this operation. There are multiple ways to achieve that, including
+mounting (`-v` option), copying (`COPY` directive or `docker cp` command),
+pick whichever suits you best.
```

Comment:
```suggestion
```suggestion
definition of your operation. The file must be available inside your
tritonserver docker image so that DALI can load it and use the newly
defined operation. This can be achieved in several ways, including
mounting (`-v` option) or copying (`COPY` directive or `docker cp` command).
```

> pick whichever suits you best.
sounds a bit too casual